docs/source/traitsui_user_manual/factories_advanced_extra.rst

```diff
@@ -172,7 +172,7 @@ with associated key bindings, and a button that invokes the key binding editor.
     class KBCodeExample ( HasPrivateTraits ):
 
         code   = Code
-        status = Str
+        status = Str()
```

Comment:
Another search-and-replace for `Code` needed?

Reply:
Yep - searched for only _one_ space.  *sigh*